src/config/dynamic-theme-fixes.config

```diff
@@ -375,6 +375,26 @@ INVERT
 
 ================================
 
+bbc.co.uk/weather
+
+CSS
+.wr-icon-weather-type__svg-background, 
+.wr-icon-rain__svg-background,
+.wr-icon-wind-direction__svg-background,
+.wr-icon-gel__svg-background {
+    opacity: 0% !important;
+}
+
+.wr-value--windspeed {
+    color: ${#dad7d2} !important;
+}
+
+.wr-c-environmental-data__icon-text {
+    color: ${#dcd9d4} !important;
+}
+
+================================
+
 bbc.com/news
 bbc.com/sport
 bbc.com/weather
```

Comment:
Make sure to remove this line

and merge the 2 invert what this section contain to above as well so it stay the same.